### DISCUSSION TITLE
History - Choose (Area|Device|Entity) drop down search box width too small

### DISCUSSION THREAD

**Discussion by chadbaldwin** [home-assistant/frontend#13648] [Other feature requests]
When viewing history on a desktop via a web browser. I find that the drop down selection/search box used to add items to the history page is not wide enough.

I usually either have to go to my entities page, find the one I want to add, then copy paste the name back on the history page. Or I have to edit the CSS to get it the width I want it to be.

Not sure if this is a browser specific thing, but I am on Chrome. The default width (for me) appears to be about 231px. I need it to be about 350px. I have _plenty_ of space on a 1080p screen for it to be wider, so it seems like this could be a simple fix to just scale it larger with larger screens. (I'm sorry, I know, nothing in CSS is simple).

The majority of my device/entity names are cut off and I can't see what's what. Sure, I could probably deduce which one I want from the icon, and the portion I _can_ see. But it seems like an extra 50% (120px) or so would be nice to have.

Example of default width:

![2022-09-07 14_28_06](https://user-images.githubusercontent.com/16479446/188953498-1ee00634-f402-4bc3-bc54-5a004439137c.png)

After changing width to 350px:

![image](https://user-images.githubusercontent.com/16479446/188953844-84ab5361-8c2f-4638-8049-e1eaefb10600.png)

**Comment by matthiasdebaat** [comment]
I agree, we need to fix this. Best if you make an issues of this with and list the pages you find the search box too small. For example on the automation editor this isn't an issue.

**Reply by ildar170975** [reply]
Similar case - https://github.com/home-assistant/frontend/issues/13671

**Comment by karwosts** [comment]
Pull request https://github.com/home-assistant/frontend/pull/14938 increases the width to 400px

**Comment by karwosts** [comment]
Resolved by #14970

**Reply by ildar170975** [reply]
You keep closing these issues - but still the list is narrow for long names.

**Reply by ildar170975** [reply]
Is it possible at least to make it a bit wider?
600px for desktops?

**Reply by karwosts** [reply]
Well I closed this because the request was for 350px, and we have added more than that. 

I don't think there needs to be many multiple duplicate issues about this open. I am aware that you would like it to be still bigger. I know Bram looked at this and settled on making it the width of the picker, which seems satisfactory for most people. 

I do know at least one issue about this is still open where you were asking for it to be wider (#14897) I will leave it open, but I'm not sure if/when it will be revisited. Hope that is satisfactory.

**Reply by karwosts** [reply]
If it was something just as simple as just replacing a 400 with a 600 in the source I would try it, but the width is currently inherited from size of the target picker, so to make it bigger would probably require more work to redesign the CSS.

**Reply by ildar170975** [reply]
> I will leave it open, but I'm not sure if/when it will be revisited. Hope that is satisfactory.

Agreed! May be some better solution will come to mind!
